package.json

```
@@ -32,6 +32,11 @@
                     "type": "object",
                     "default": null,
                     "description": "Specifies extensions for different shader stages. Keys shall be extensions (for example '.vs.glsl')."
+                },
+                "glsl-linter.validatorArgs" : {
+                    "type": "string".
```

I think there's a typo in this line
```suggestion
                    "type": "string",
```